lib/metadata_json_lint.rb

```diff
@@ -77,6 +77,11 @@ def contains_invalid_escape?(content)
   end
   module_function :contains_invalid_escape?
 
+  def contains_invalid_newline?(content)
```

Comment:
Perhaps make it a bit more matching the actual implementation?
```suggestion
  def misses_newline_at_end?(content)
```